lib/zoom/client.rb

```diff
@@ -35,14 +35,20 @@ def headers
 
     def oauth_request_headers
       {
-        'Authorization' => "Basic #{auth_token}"
-      }.merge(headers)
+        'Authorization' => "Basic #{auth_token}",
+        'Accept' => 'application/json',
+        'Content-Type' => 'application/x-www-form-urlencoded',
+      }
     end
 
-    def request_headers
+    def bearer_authorization_header
       {
         'Authorization' => "Bearer #{access_token}"
-      }.merge(headers)
+      }
+    end
+
+    def request_headers
+      bearer_authorization_header.merge(headers)
```

Comment:
This seems like a refactor that is outside the scope of this change. I would think we should just use `bearer_authorization_header` and not use `request_headers` at all. I agree it makes sense to differentiate between the oauth headers and the "standard" headers more precisely.

Reply:
@joelmichael Do you mean use `bearer_authorization_header` without merging it with `headers`?